Lab_3.sc

```diff
@@ -0,0 +1,115 @@
+class Lab_3 {
+  def podziel(list: List[Int]): (List[Int], List[Int]) = {
+    def pom1(list: List[Int]): List[Int] =
```

Comment:
stosuj prosze konsekwentnie angielskie nazwy funkcji i to mówiące o tym co dana funkcja robi. Tu na przyklad pom1 pom2 nie mam pojęcia co autor miał na myśli